mix.exs

```diff
@@ -8,34 +8,23 @@ defmodule DllLoaderHelper.MixProject do
       version: "0.1.0",
       elixir: "~> 1.13",
       start_permanent: Mix.env() == :prod,
-      compilers: [:elixir_make] ++ Mix.compilers(),
+      compilers: compilers() ++ Mix.compilers(),
       source_url: @github_url,
       description: description(),
       package: package(),
-      make_executable: make_executable(),
-      make_makefile: make_makefile(),
       deps: deps()
     ]
   end
 
-  def make_executable() do
+  defp compilers() do
     case :os.type() do
-      {:win32, _} -> "nmake"
-      _ -> "make"
-    end
-  end
-
-  def make_makefile() do
-    case :os.type() do
-      {:win32, _} -> "Makefile.win"
-      _ -> "Makefile.unix"
+      {:win32, _} -> [:elixir_make]
```

Comment:
This is the tricky to only run `elixir_make` on Windows. :) `nmake` and `Makefile.win` are the defaults then!